### PR TITLE
Sync install and SHE docs with the latest RHV 4.3.8 content

### PR DESCRIPTION
**Pull request by pmkovar**
Related issue #2086

Follow up for #2129 

Changes proposed in this pull request:

This is a complete import of restructured install docs for SHE and standard deployments. 

* Sync install and SHE docs with the latest RHV 4.3.8 content

* Modularize install and SHE docs by breaking them up into multiple
titles.

* Update links to the docs and set up redirects.

@sandrobonazzola The reference links to other docs might need further tweaking but at this early point of syncing docs with downstream, I rather went for a quick fix.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @pmkovar

This pull request needs review by: @sandrobonazzola @duck-rh 

cc @rolfedh

### CODE REVIEW

**Comment by pmkovar** [comment]
@duck-rh Note that even when I move the shared `adoc` files outside the `source` tree, I still see individual files built under the `source/documentation/common` symlink.

As a workaround, perhaps we could just delete these unneeded `common/*.html` files as part of the deployment script?

File: asciidoc-common/admin/proc-Removing_Red_Hat_Virtualization_Manager.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+----
+During execution engine service will be stopped (OK, Cancel) [OK]:
+ovirt-engine is about to be removed, data will be lost (OK, Cancel) [Cancel]:OK
+----
++
+. Remove the {virt-product-fullname} packages:
++
+[options="nowrap" subs="normal"]
+----
+# yum remove rhvm* vdsm-bootstrap
```
Comment (sandrobonazzola) [inline asciidoc-common/admin/proc-Removing_Red_Hat_Virtualization_Manager.adoc]: I think here a conditional is needed, rhvm is not available on ovirt. I don't recall I ever heard about vdsm-bootstrap. @michalskrivanek ?

File: asciidoc-common/install/snip-RHVM_repo_steps.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
++
+[options="nowrap" subs="normal"]
+----
+# subscription-manager repos \
+    --disable='*' \
+    --enable=rhel-7-server-rpms \
+    --enable=rhel-7-server-supplementary-rpms \
+    --enable=rhel-7-server-rhv-4.3-manager-rpms \
+    --enable=rhel-7-server-rhv-4-manager-tools-rpms \
+    --enable=rhel-7-server-ansible-2-rpms \
```
Comment (sandrobonazzola) [inline asciidoc-common/install/snip-RHVM_repo_steps.adoc]: if this is not already under conditional, above repos are fine also for oVirt, except the rhel-7-server-rhv-4.3-manager-rpms and rhel-7-server-rhv-4-manager-tools-rpms which are not available with a RHEL subscripton.

File: asciidoc-common/install/snip-RHEL_repo_steps.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
++
+.. Configure the repositories:
++
+[options="nowrap" subs="normal"]
+----
+# subscription-manager repos \
+    --disable='*' \
+    --enable=rhel-7-server-rpms \
+    --enable=rhel-7-server-rhv-4-mgmt-agent-rpms \
+    --enable=rhel-7-server-ansible-2-rpms
```
Comment (sandrobonazzola) [inline asciidoc-common/install/snip-RHEL_repo_steps.adoc]: if this section is not already under conditional doc, rhel-7-server-rhv-4-mgmt-agent-rpms is not avilable with standard RHEL subscription so it's not available for ovirt users.

File: asciidoc-common/install/proc-Installing_the_Websocket_Proxy_on_a_different_host.adoc [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+. Run the `engine-setup` command to configure the websocket proxy.
++
+[options="nowrap" subs="normal"]
+----
+# engine-setup
+----
++
+[NOTE]
+====
+If the `rhvm` package has also been installed, choose `No` when asked to configure the {engine-name} (`Engine`) on this host.
```
Comment (sandrobonazzola) [inline asciidoc-common/install/proc-Installing_the_Websocket_Proxy_on_a_different_host.adoc]: this note should be on rhv doc, on ovirt we have ovirt-engine instead.

File: asciidoc-common/admin/proc-Removing_Red_Hat_Virtualization_Manager.adoc (hunk 1 above)
Comment (michalskrivanek) [reply asciidoc-common/admin/proc-Removing_Red_Hat_Virtualization_Manager.adoc]: wasn't it support for legacy rhvh? obsolete for a long time....

File: asciidoc-common/install/snip-RHVM_repo_steps.adoc (hunk 2 above)
Comment (michalskrivanek) [reply asciidoc-common/install/snip-RHVM_repo_steps.adoc]: I'm not sure about ansible either

File: asciidoc-common/admin/proc-Removing_Red_Hat_Virtualization_Manager.adoc (hunk 1 above)
Comment (pmkovar) [reply asciidoc-common/admin/proc-Removing_Red_Hat_Virtualization_Manager.adoc]: Added a conditional.

Let me know if you think we don't really need `vdsm-bootstrap` for RHV either.

File: asciidoc-common/install/snip-RHVM_repo_steps.adoc (hunk 2 above)
Comment (pmkovar) [reply asciidoc-common/install/snip-RHVM_repo_steps.adoc]: This is actually conditionalized already, so it seems like that file is unneeded.

File: asciidoc-common/install/snip-RHEL_repo_steps.adoc (hunk 3 above)
Comment (pmkovar) [reply asciidoc-common/install/snip-RHEL_repo_steps.adoc]: This is actually conditionalized already, so it seems like that file is unneeded.

File: asciidoc-common/install/proc-Installing_the_Websocket_Proxy_on_a_different_host.adoc (hunk 4 above)
Comment (pmkovar) [reply asciidoc-common/install/proc-Installing_the_Websocket_Proxy_on_a_different_host.adoc]: Added a conditionalized note for ovirt.

**Comment by pmkovar** [comment]
> @pmkovar I looked into it and adding`documentation/common` in the `exclude:` list into `_config.yml` solved the problem; the symlink is then not necessary anymore (tested too).

@duck-rh Thanks for looking into it. I've moved the files around and `adoc` files now render fine, but there is a problem with images not showing if placed underneath the excluded directory. 

Since the images in this PR are not really all that important for the docs, I chose to remove them for now. We can try to address that problem some time later.

**Comment by duck-rh** [comment]
@pmkovar I merged  #2129 so we can merge this one after rebase. I'll look into the image problem. We can merge this one too and solve this in a later PR if you wish.

**Comment by duck-rh** [comment]
@pmkovar `documentation/common/**/*.adoc` works and keeps all the other files

**Comment by pmkovar** [comment]
> @pmkovar `documentation/common/**/*.adoc` works and keeps all the other files

@duck-rh Thanks, it works! Re-added images and rebased. 

I think we can merge now.

**Comment by pmkovar** [comment]
@duck-rh Apologies, restored the files and squashed, too.

**Pull request merged**